### ISSUE TITLE
Add fiddle into readme

### ISSUE THREAD

**Issue by hwclass** [tictail/tide#26]
Hi guys,

I have just seen your work while I was looking at the alternative state managers to Redux and found Tilde, thanks for your great job.

Just thought that maybe having a fiddle link to direct the visitors to try online would be nice? What do you think?

Cheers,

Baris

**Comment by eldh** [comment]
Yeah that's a great idea, we'll try to fix that soon. Also, there's still a few changes that we made in v2.0 that hasn't really made it into the docs yet.
